themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -593,6 +593,8 @@ consent.option=consent
 login.option=login
 select-account.option=select_account
 prompt.tooltip=Specifies whether the Authorization Server prompts the End-User for reauthentication and consent.
+supports-prompt-none=Support prompt=none
```

Comment:
This is another topic I wanted a feedback on. To me "Support prompt=none" is a good name for the new configuration but I'm open to suggestions.

Reply:
Suggested in description